### PR TITLE
[gdb] Add makefiles for easier compiling

### PR DESCRIPTION
**Pull request by cbiesinger**

### CODE REVIEW

File: server/JsDbg.Gdb/Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+PREFIX=/usr/local
+DOTNET=$(HOME)/dotnet/dotnet
+
+PUBLISH=bin/Release/netcoreapp2.1/linux-x64/publish
```
Comment (dlibby-) [inline server/JsDbg.Gdb/Makefile]: Is `netcoreapp2.1` a constant, or should we get it from the version somehow? Looks like the current download page points to 2.2 - does it still produce output at netcoreapp2.1?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+all:
```
Comment (dlibby-) [inline Makefile]: My Makefile knowledge is very limited (so this is more for my understanding) - do we need this all target or would the % one take care of this as well?

File: server/JsDbg.Gdb/Makefile (hunk 1 above)
Comment (dlibby-) [reply server/JsDbg.Gdb/Makefile]: Looks like this matches the existing path from the python script, so I'm guessing not an issue (or at least can be tackled in a separate change).

Comment (cbiesinger) [reply server/JsDbg.Gdb/Makefile]: Fairly sure this comes from the TargetFramework in the .csproj file -- I do have the dotnet sdk 2.2 and it does output into netcoreapp2.1.

File: Makefile (hunk 2 above)
Comment (cbiesinger) [reply Makefile]: Unfortunately it seems we need this target if we want "make" without an explicit target to work.

**Pull request merged**